packages/language-server/src/__test__/completion.test.ts

```diff
@@ -137,6 +137,36 @@ suite('Completions', function () {
     kind: CompletionItemKind.Field,
   }
 
+  suite('SCALAR LISTS', () => {
+    const fieldsPropertyEmptyList = {
+      label: '[]',
+      kind: CompletionItemKind.Value,
+    }
+
+    const functionDbGenerated = {
+      label: 'dbgenerated()',
+      kind: CompletionItemKind.Function,
+    }
+
+    test('@default', () => {
+      const scalarTypes = ['String', 'color', 'Int', 'Float', 'Boolean', 'DateTime'] as const
```

Comment:
Where is `color` from here? it's not in the schema

Reply:
I got it straight out of this comment: https://github.com/prisma/language-tools/issues/1159#issuecomment-1160213872

But now I realize I forgot to add

```prisma
        enum color {
          RED
          GREEN
          BLUE
        }
```

to the schema 👍🏻